core/node/crypto/config.go

```diff
@@ -20,15 +20,44 @@ import (
 )
 
 var (
-	// StreamReplicationFactorKey is the key for how often a stream is replicated over nodes
-	StreamReplicationFactorKey = newChainKeyImpl("stream.replication.factor")
+	StreamMediaMaxChunkCountConfigKey            = newChainKeyImpl("stream.media.maxChunkCount", uint64Type)
+	StreamMediaMaxChunkSizeConfigKey             = newChainKeyImpl("stream.media.maxChunkSize", uint64Type)
+	StreamRecencyConstraintsAgeSecondsConfigKey  = newChainKeyImpl("stream.recencyConstraints.ageSeconds", uint64Type)
+	StreamRecencyConstraintsGenerationsConfigKey = newChainKeyImpl("stream.recencyConstraints.generations", uint64Type)
+	// StreamReplicationFactorConfigKey is the key for how often a stream is replicated over nodes
+	StreamReplicationFactorConfigKey                = newChainKeyImpl("stream.replicationFactor", uint64Type)
+	StreamDefaultMinEventsPerSnapshotConfigKey      = newChainKeyImpl("stream.defaultMinEventsPerSnapshot", uint64Type)
+	StreamMinEventsPerSnapshotUserInboxConfigKey    = newChainKeyImpl("stream.minEventsPerSnapshot.a1", uint64Type)
+	StreamMinEventsPerSnapshotUserSettingsConfigKey = newChainKeyImpl("stream.minEventsPerSnapshot.a5", uint64Type)
+	StreamMinEventsPerSnapshotUserConfigKey         = newChainKeyImpl("stream.minEventsPerSnapshot.a8", uint64Type)
+	StreamMinEventsPerSnapshotUserDeviceConfigKey   = newChainKeyImpl("stream.minEventsPerSnapshot.ad", uint64Type)
+	StreamCacheExpirationConfigKey                  = newChainKeyImpl("stream.cacheExpiration", uint64Type)
```

Comment:
For all time values please add unit to the name: in normal config it's set as 5s or such, so it's obvious what the value is. Here it's set as uint, so without explicit probability of config bugs is going to be high.

(Also, nanos is rather extreme unit to operate with)